README.md

```diff
@@ -1,14 +1,24 @@
 Nix, the purely functional package manager
 ------------------------------------------
 
-Nix is a new take on package management that is fairly unique. Because of it's
+Nix is a new take on package management that is fairly unique. Because of its
 purity aspects, a lot of issues found in traditional package managers don't
 appear with Nix.
 
+*Darwinix* is an OSX-specific port of nix that integrates more tightly with Xcode,
+avoiding a host of problems introduced by the fact that the vast majority of OSX's system
+libraries are closed-source.
+
 To find out more about the tool, usage and installation instructions, please
 read the manual, which is available on the Nix website at
 <http://nixos.org/nix/manual>.
 
+## Installing
+
+``` bash
+$ nix-env -f release.nix -iA build.x86_64-darwin
+```
```

Comment:
We shouldn't duplicate the manual here, so please don't add installation instructions here.